### PR TITLE
Add favicon SVGs

### PR DESCRIPTION
**Pull request by projectyang**
Adds in favicon SVGs created for taskcluster UI and updates README description.

### CODE REVIEW

File: source/logo/faviconLogoRunning.svg [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
+</defs>
+</svg>
```
Comment (helfi92) [inline source/logo/faviconLogoRunning.svg]: When I try to load this into sketch, I get

Screen Shot 2019-04-26 at 11 06 24 AM

I think it's missing the image svg inside.

File: source/logo/faviconLogoRunning.svg [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
...w+l0E+fnNls66DgHyFAwSQH6DaDjhhAACdBJk6jaQX2qirGeRAAK0mErlmpBf5QDYvhgBBFgMdRsbIb82cqLKNAQQYBqOLlZBfi5ipIkZBBDgDFieL0V+ntOltyECCHCITKDPkV+gsGn1FAEEeApHvDfIL17mdHxMAAEeswj3CvmFi5yGNwggwA0gUd4ivyhJ0+c2AghwGx2n55Cf02BpazYBBDgbWds3IL+286P6tAQQYFqepldDfqbjobgKBBBgBeg1tkR+Naizp3UCCNB6QgnqQ34JILKESwII0GWsx00hv2MWvILAJoGdzQ9474vAlx//p9u9cs1XU3QDgUQE+AkwEUiryzx+9LB7vP/QannUBYGqBBBgVfxlNkeCZTizS3sEEGB7mS2qGAkuwsZNzgkgQOcBn2wPCZ6kwWsI8EuRwj0DSDBc5DS8hQA/AW6B4/UUEvSaLH3NJYAA5xJzcj0SdBIkbawigABX4Wv7ZiTYdn5Uv54AAlzPsOkVkGDT8VH8SgIIcCVAD7cjQQ8p0sMSAghwCTWH9yBBh6HS0igBBDiKKM4FSDBO1nTaE0CAPAmnCCDBUzh445wAAnQe8JL2kOASatzTIgEE2GJqBWpGggUgs0V1AgiwegR2C0CCdrOhsjQEEGAajm5XQYJuo6WxAwIIkMdglAASHEXEBY0SQICNBle6bCRYmjj7lSCAAEtQdrIHEnQSJG0cEUCARyh4MYUAEpxCiWtaIYAAW0nKUJ1I0FAYlLKKAAJchS/uzUgwbvaeOkeAntIs3AsSLAyc7ZITQIDJkcZaEAnGyttbtwjQW6IV+kGCFaCzZRICCDAJRhZBgjwDLRJAgC2mZrRmJGg0GMoaJIAAB9FwYgkBJLiEGvfUIoAAa5F3vC8SdByus9YQoLNArbSDBK0kQR3bCCDAbXQ4t4oAElyFj5sLEECABSBH3gIJRk7ffu8I0H5GzVeIBJuP0G0DCNBttLYaQ4K28qCangAC5EkoRgAJFkPNRhMJIMCJoLgsDQEkmIYjq6QhgADTcGSVGQSQ4AxYXJqVAALMipfFhwggwSEyfF6SAAIsSZu9ThHY+/Zet/fNvVOf8QYCJQnslNyMvSCgBB7t/a/b//mPfHb5mRf0FDMEihHgJ8BiqNlICaj89P0DfhJUFMyFCfwf8l/A6WDGZiUAAAAASUVORK5CYII="/>
```
Comment (helfi92) [inline source/logo/faviconLogoRunning.svg]: Should this be width="32" and height="32"?

File: source/logo/faviconLogoRunning.svg (hunk 1 above)
Comment (projectyang) [reply source/logo/faviconLogoRunning.svg]: Hmmm, I'm not sure what's the issue. They seem to be loading fine for me - tested on 2 computers.

EDIT: Whoa, no definitely off from the conversion. I used figma, hold up let me check. 

Repo SVGs:
Screen Shot 2019-04-26 at 11 21 35 AM

SVGs I sent in email:
Screen Shot 2019-04-26 at 11 25 42 AM

Comment (projectyang) [reply source/logo/faviconLogoRunning.svg]: Apologies, I didn't realize that SVGs are not standardized in a way...

The tool I used originally was Figma, which seems to have SVG compatibility issues with other design tools. 

https://www.figma.com/blog/with-figmas-new-svg-exports-less-more/
https://spectrum.chat/figma/general/svg-export-issues~9053ed2f-9ec7-4116-80c3-1a9207511905

I'll recreate in sketch.

**Comment by projectyang** [comment]
Recreated the SVGs in sketch. 

One major change I've done was replacing the logo image files with just the original logo SVG. This was done to improve consistency between each logo (new SVGs in the first row). I've included the old SVGs in the pull for now if you want to review.

Screen Shot 2019-04-26 at 1 14 32 PM

The difference in a generated PNG is very minimal - however, I can open up another pull request to replace the ones we just merged with fresh ones. Let me know.

Again, thanks for the patience here. As a developer, this was a fun deep dive into the world of design...

File: source/logo/faviconLogo.svg [hunk 3]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (helfi92) [inline source/logo/faviconLogo.svg]: Maybe it makes more sense to put the favicon logos under `source/favicons/`. Also, feel free to put the pngs under `media.taskcluster.net/favicons/`.

**Review by helfi92 — CHANGES_REQUESTED** [review]
Looks good :) I think it should be okay to remove the old favicons from this pull-request.

**Review by helfi92 — APPROVED** [review]
Looks good to me. Thanks @projectyang :)

**Pull request merged**